daisy/step_includeworkflow.go

```diff
@@ -81,6 +81,16 @@ func (i *IncludeWorkflow) populate(ctx context.Context, s *Step) dErr {
 	}
 	substitute(reflect.ValueOf(i.Workflow).Elem(), strings.NewReplacer(replacements...))
 
+	// We do this here, and not in validate, as embedded startup scripts could
+	// have what we think are daisy variables.
+	if err := i.Workflow.validateVarsSubbed(); err != nil {
+		return err
```

Comment:
instead of doing the if err := __; err != nil { return err } thing, you can do errs = addErrs(errs, ___). If you want.

Reply:
Same comment for other changes, too.

Reply:
These are pretty much hard fails, that's why these functions don't already do that. For example failure to populate something could cause a string of other failures, so better to just return the error when you get it.